impeller/entity/contents/atlas_contents.cc

```diff
@@ -396,8 +397,7 @@ bool AtlasTextureContents::Render(const ContentContext& renderer,
     return true;
   }
 
-  Command cmd;
-  DEBUG_COMMAND_INFO(cmd, "AtlasTexture");
+  pass.SetCommandLabel("AtlasTexture");
```

Comment:
FYI this is still incurring a copy because the parameter takes a `std::string`.

Reply:
https://github.com/flutter/engine/blob/main/impeller/renderer/render_pass.h#L74 This takes a string view.

Reply:
`RenderPass::OnSetLabel` takes an `std::string`, it doesn't hit that?

Reply:
Maybe we should rename SetCommandLabel at some point since the concept of Command has been eliminated.

Reply:
Yeah, I think that it would be better if the RenderPass label was provided during construction instead of via a Setter. Then we just need "SetLabel"